src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -427,12 +454,10 @@ const IssueForm = (): JSX.Element | null => {
               </h5>
             }
             unitIcon={<WrappedTokenLogoIcon width={20} />}
-            value={wrappedTokenAmount.toHuman(8)}
+            role='total-receiving-amount'
```

Comment:
This isn't the correct use of the role attribute (it's not intended to be used for arbitrary values). If you have a look at the testing library documentation there's a link to a list of roles:
https://testing-library.com/docs/queries/about/#priority

The way it's intended to be used is by querying by role and also by name, e.g. if you have multiple elements assigned the role of `button` then you differentiate between them by querying for a name as well: `getByRole('button', {name: /submit/i})`.

Ideally we wouldn't need to add any additional attributes to our code to write the tests, but if that's proving difficult then `data-testid` can be used, although this should only be uses as a fallback option.

Reply:
Thank you for your feedback. I will adjust the code accordingly.

Reply:
Hi @tomjeatt 
I've investigated how to fix this query issue and, in my opinion, using `data-testid` for now is an acceptable approach given how `PriceInfo` component is structured.
I tried to update `PriceInfo` component for better accessibility but have not been able to figure out a better one.